parsec/backend/invite.py

```diff
@@ -161,16 +647,18 @@ async def api_invite_new(self, client_ctx, msg):
             if client_ctx.profile != UserProfile.ADMIN:
                 return invite_new_serializer.rep_dump({"status": "not_allowed"})
 
-            # TODO: implement send email feature
-            if msg["send_email"]:
-                return invite_new_serializer.rep_dump({"status": "not_implemented"})
-
             invitation = await self.new_for_user(
                 organization_id=client_ctx.organization_id,
                 greeter_user_id=client_ctx.user_id,
                 claimer_email=msg["claimer_email"],
             )
 
+            # TODO: implement send email feature
+            # moved this if block here to have infos on the greeter id
```

Comment:
No need for this comment (when reading the code we don't remember the block used to be somewhere else).
This is the kind of discussion that should happen in the code review though.